source/Planet.h

```diff
@@ -68,6 +68,10 @@ class Planet {
 	// have the "uninhabited" attribute).
 	bool IsInhabited() const;
 	
+	// Check if the security of this planet has been changed from the default so
+	// that we can check if an uninhabited world should fine the player.
+	bool SecurityChanged() const;
```

Comment:
I think a better name is in order, e.g. `HasCustomSecurity` (default false) or `HasDefaultSecurity` (default true).

Reply:
I'll go with the former.